modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -83,21 +84,53 @@ spec:
           name: cloud-credentials
         default: true
         objectStorage:
-          bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          bucket: my-bucket
+          prefix: velero
         provider: aws
   configuration:
     restic:
-      enable: <true_or_false>
+      enable: false
     velero:
+       itemOperationSyncFrequency: "10s"
       defaultPlugins:
         - openshift
         - aws
         - csi
+        - vsm
+       resourceTimeout: 10m <1>
+  features:
+    dataMover:
+      credentialName: restic-secret
+      enable: true
+      maxConcurrentBackupVolumes: "3" <2>
+      maxConcurrentRestoreVolumes: "3" <3>
+      pruneInterval: "14"  <<<<<<<--------------------  pruneInterval <4>
+      volumeOptions:  <5>         <------- volumeOptions
+        sourceVolumeOptions:
+          accessMode: ReadOnlyMany
+          cacheAccessMode: ReadWriteOnce
+          cacheCapacity: 2Gi
+          moverSecurityContext: true
+        destinationVolumeOptions:
+          storageClass: other-storageclass-name
+          cacheAccessMode: ReadWriteMany
+          moverSecurityContext: true
+  snapshotLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-west-2
+        provider: aws
+
 ----
-<1> Add the Restic secret name from the previous step. If this is not done, the default secret name `dm-credential` is used.
+<1> Optional: Specify the minutes before timeouts in Velero, based on infrastructure and performance.
+<2> Specify the upper limit of the number of snapshots allowed to be queued for backup.
+<3> Specify the upper limit of the number of snapshots allowed to be queued for restore.
+<4> Specify the number of days between running restic pruning on the repository. The prune operation repacks the data to free space, but it can also generate significant I/O traffic as a part of the process. Setting this option allows a trade-off between storage consumption (from no longer referenced data) and access costs.
+<5> Configure the VolumeSync backup and restore options.
+
```

Comment:
Is this a sufficient description of the new features for the user in the module? Should the expanded description appear in the release notes? If an expanded description is required in this module, where should it appear?

Reply:
<1> is not specific to data mover, but to OADP in general. If we want to keep this doc only for the new data mover features, I would suggest removing resourceTimeout from this example.

Reply:
I think we should mention here that this fields are optional

Reply:
@PrasadJoshi12 which field/s are you referring to? <3>, <4> or <5>? Please specify?

Reply:
<1> Optional: if not specified then default value will be 10. 
<2> Optional:  if not specified then default value will be 10. 
<3> Optional
<4> Optional

Reply:
@PrasadJoshi12 @eemcmullan are 1- 4 optional because they are only relevant for OADP 1.2?

Reply:
They are new to 1.2, but they are still optional in 1.2 because datamover does not necessarily need these values to complete